### PR TITLE
Poll text margins need to be changed [YEP-64]

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@smn @bruskiza @miltontony ready for review :)

File: tuneme/static/css/styles.css [hunk 1]
```diff
@@ -108,7 +108,7 @@ label {font:normal 15px 'Quicksand', Arial, Helvetica, sans-serif;}
 
     /* Radios */
     input[type='radio'] {width:auto; padding:0px; margin:0 5px 0 0; font-size:15px;}
-    .radios label { font-size:15px; min-width:75px; font-weight:normal;}
+    .radios label { font-size:15px; min-width:75px; font-weight:normal; display: inline-flex; width: 90%;}
```
Comment (miltontony) [inline tuneme/static/css/styles.css]: I doubt the support for `flex` on mobile devices. Can you use `inline-block` instead?

Comment (smarzban) [inline tuneme/static/css/styles.css]: oh ok, sure :)

**Comment by smarzban** [comment]
@miltontony fixed

**Comment by bruskiza** [comment]
:+1:

**Comment by miltontony** [comment]
:+1:

**Pull request merged**